.env.sample

```diff
@@ -0,0 +1,9 @@
+# create a .env.local file, copy and fill the following data:
```

Comment:
Why this file is getting added, I removed it by adding it in .gitignore

Reply:
This is just a sample env file and doesn't contain any actual variables. It is used to guide other developers who might want to run the repository locally